testsuite/MDAnalysisTests/core/test_topologyattrs.py

```diff
@@ -204,9 +220,18 @@ def test_set_residues(self):
 
 
 class TestResnames(TestResidueAttr):
-    values = np.array(['ARG', 'LYS', 'VAL', 'POPG'], dtype=np.object)
+    values = np.array(['VAL', 'LYS', 'VAL', 'POPG'], dtype=np.object)
     attrclass = tpattrs.Resnames
 
+    def test_get_named_residue(self):
```

Comment:
So the `Attr` can define methods that can transplanted onto a particular group.  So here the transplant is defined:

https://github.com/MDAnalysis/mdanalysis/blob/issue-363/package/MDAnalysis/core/topologyattrs.py#L968

And when Universe loads a topology, it processes these and fills the Group classes with the methods:

https://github.com/MDAnalysis/mdanalysis/blob/issue-363/package/MDAnalysis/core/universe.py#L493